src/lib.rs

```diff
@@ -395,6 +401,42 @@ pub trait Context {
 
     unsafe fn uniform_1_i32(&self, location: Option<Self::UniformLocation>, x: i32);
 
+    unsafe fn uniform_2_i32(&self, location: Option<Self::UniformLocation>, x: i32, y: i32);
+
+    unsafe fn uniform_3_i32(&self, location: Option<Self::UniformLocation>, x: i32, y: i32, z: i32);
+
+    unsafe fn uniform_4_i32(&self, location: Option<Self::UniformLocation>, x: i32, y: i32, z: i32, w: i32);
+
+    unsafe fn uniform_1_i32_v(&self, location: Option<Self::UniformLocation>, v: &mut [i32; 1]);
```

Comment:
I've been using `_slice` instead of `_v` so far, what do you think?

Reply:
`_slice` sounds more natural! I'll fix them.